docs/index.rst

```diff
@@ -47,3 +48,57 @@ Advanced
 
    api
    development
+
+Reporting Issues
+================
+
+If you have found a bug in Regions please report it by creating a
+new issue on the `Regions GitHub issue tracker
+<https://github.com/astropy/regions/issues>`_.
+
+Please include an example that demonstrates the issue that will allow
+the developers to reproduce and fix the problem. You may be asked to
+also provide information about your operating system and a full Python
+stack trace.  The developers will walk you through obtaining a stack
+trace if it is necessary.
+
+Astropy Regions uses a package of utilities called `astropy-helpers
+<https://github.com/astropy/astropy-helpers>`_ during building and
+installation.  If you have any build or installation issue mentioning
+the ``astropy_helpers`` or ``ah_bootstrap`` modules please send a
+report to the `astropy-helpers issue tracker
+<https://github.com/astropy/astropy-helpers/issues>`_.  If you are
+unsure, then it's fine to report to the main Regions issue tracker.
+
+
+Contributing
+============
+
+Like the `Astropy`_ project, Regions is made both by and for its
+users.  We accept contributions at all levels, spanning the gamut from
+fixing a typo in the documentation to developing a major new feature.
+We welcome contributors who will abide by the `Python Software
+Foundation Code of Conduct
+<https://www.python.org/psf/codeofconduct/>`_.
+If you have a feature request or would like to contribute to ``regions``,
+please go here: https://github.com/astropy/regions/
+
+Regions follows the same workflow and coding guidelines as
+`Astropy`_.  The following pages will help you get started with
+contributing fixes, code, or documentation (no git or GitHub
+experience necessary):
+
+* `How to make a code contribution <http://astropy.readthedocs.io/en/stable/development/workflow/development_workflow.html>`_
+
+* `Coding Guidelines <http://docs.astropy.org/en/latest/development/codeguide.html>`_
+
+* `Try the development version <http://astropy.readthedocs.io/en/stable/development/workflow/get_devel_version.html>`_
+
+* `Developer Documentation <http://docs.astropy.org/en/latest/#developer-documentation>`_
+
+
+Get Help
+========
+
+Besides github, You can `get help`_ from the community in a number of ways.
+There is also a slack channel for regions.
```

Comment:
"...hosted under the main astropy slack."

Reply:
While I agree that it would be good to have non-developer forums here, such as the astropy-users list or even stackoverflow, I don't want to see facebook mentioned.  I'm not on it and I don't want to encourage others to use it.

Reply:
That's understandable. It's interesting to see how and why that forum works, but it does, and seems much friendlier than SO (though I think SO is fine, too if your expectations well aligned with their original purpose).
What about linking to the astropy help page, then? http://www.astropy.org/help.html

Reply:
It is already linked( sorry, i forgot to mention it earlier)